### ISSUE TITLE
no-spaced-func rule still has false positives

### ISSUE THREAD

**Issue by michaelficarra** [eslint/eslint#412]
``` js
f(0, (1))
```

I'll be making a change to how the rule works, and hopefully we'll get it right once and for all.
